File: Codes/gracekoo/58_length-of-last-word.py
# -*- coding: utf-8 -*-
# @Time: 2020/3/1 18:01
# @Author: GraceKoo
# @File: 58_length-of-last-word.py
# @Desc:https://leetcode-cn.com/problems/length-of-last-word/


class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        if not s:
            return 0
        return len(s.split()[-1])


so = Solution()
print(so.lengthOfLastWord("Hello World"))

File: Codes/yore/58_length_of_last_word/58_length_of_last_word.java
class Solution {
    public int lengthOfLastWord(String s) {
        if (s.length() == 0) {
            return 0;
        }
        String[] array = s.split(" ");
        int length = array.length;
        if (length > 0) {
            String str = array[length - 1];
            return str.length();
        }
        return 0;
    }
}